message_converter/src/converter_node.cpp

```diff
@@ -244,8 +234,16 @@ int main(int argc, char **argv)
     // Values in comments are from the parking lot test
     // double home_latitude = 42.3711071377; 
     // double home_longitude = -71.2168400352; 
-    // double home_altitude = -15.3775837652; 
+    // double home_altitude = -15.3775837652;
+    // double home_latitude = 40.8502544178; 
+    // double home_longitude = 14.3080716592; 
+    // double home_altitude = 77.7699139992;     
```

Comment:
maybe you want to drop these since you're loading them from ros params now?

Reply:
also kudos on using ROS params instead of hard coding

Reply:
Fixed in https://github.com/Humatics/ethz_piksi_ros/pull/1/commits/c275451451b84a2384550d93f7675d8dbdde6f24